### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by D0N2k** [exodus4d/pathfinder#618]
Hi,

I installed pathfinder on a vserver with apache. 
Now I got the problem that sometimes members are logged out with Error 500: Internal Server Error.

Sometimes with this error message:
`Internal Server Error
Fatal error: Unknown: Unable to load dynamic library 'zmq.so' (tried: /opt/plesk/php/7.2/lib/php/modules/zmq.so (/opt/plesk/php/7.2/lib/php/modules/zmq.so: cannot open shared object file: No such file or directory), /opt/plesk/php/7.2/lib/php/modules/zmq.so.so (/opt/plesk/php/7.2/lib/php/modules/zmq.so.so: cannot open shared object file: No such file or directory))`

php -m says zmq is installed and loaded.

Thanks in advance.

**Comment by exodus4d** [comment]
This looks like an Issue I had after upgradint  7.0 -> 7.2 . For each PHP version you need to install a suitable version for each library. I would try a re-install or this PHP libraries over _PECL_ (like I do):

1. check _PECL_ version
```
$ pecl version
PEAR Version: 1.10.4
PHP Version: 7.0.25
Zend Engine Version: 3.0.0
```
2. install ZMQ PHP extension
``````

**Comment by D0N2k** [comment]
Hey sorry for the delay.

I installed it the way you told me php -m says zmq and event is loaded. but php info says it isn't.

My problem is that's the vserver of my corp. It runs Plesk with 5 different php versions. I edited the php.ini listed within php info.

Thats another error i ran into `Internal Server Error
Fatal error: Unknown: zmq: Unable to initialize module Module compiled with module API=20151012 PHP compiled with module API=20170718 These options need to match`

**Comment by D0N2k** [comment]
Okay I solved the Issue with zmq not loading.

Now I got 3 new errors within my logs:
`Got error 'PHP message: HTTP 404 (GET /favicon.ico)\nPHP message: [index.php:23] Base->run()\n'`
`Got error 'PHP message: Fatal error: Module 'zmq' already loaded\nPHP message: [Unknown:0] \n'`
`Got error 'PHP message: HTTP 404 (GET /ws/map/update)\nPHP message: [index.php:23] Base->run()\n'`

Now I commented zmq.so within my php.ini seems to work fine for now. But I dont know where those 404 come from.
